res/ci/win_build.sh

```
@@ -44,7 +44,10 @@ fi
 
 rm -rf bin && mkdir -p bin
 
-for f in $(find src -type f -name '*.cpp'); do compile "bin/$(basename $f).o" "$f"; done
+for f in $(find src -type f -name '*.cpp'); do
```

Shellcheck (0.11.0)

[warning] For loops over find output are fragile. Use find -exec or a while read loop.

(SC2044)